po/fi.po

```diff
@@ -0,0 +1,688 @@
+msgid "failed to download sources"
+msgstr "ei pystytty latamaan lähteitä"
+
+#: src/install.rs:230 src/install.rs:246 src/install.rs:254
+msgid "failed to build"
+msgstr "ei pystytty rakentamaan"
+
+#: src/install.rs:233
+msgid "parsing pkg list..."
+msgstr "jäsennetään pkg listaa..."
+
+#: src/install.rs:261 src/install.rs:1462
+msgid "{}-{} is up to date -- skipping build"
+msgstr "{}-{} on ajantasalla -- ohitetaan rakennus"
+
+#: src/install.rs:333 src/query.rs:50 src/lib.rs:84
+msgid "error:"
+msgstr "virhe:"
+
+#: src/install.rs:334
+msgid "could not get news"
+msgstr "ei voitu hakea uutisia"
+
+#: src/install.rs:339 src/install.rs:521 src/install.rs:716 src/install.rs:848
+msgid "Proceed with installation?"
+msgstr "Jatka asentamista?"
+
+#: src/install.rs:356
+msgid "no targets specified (use -h for help)"
+msgstr "yhtään kohdetta ei ole määritelty (-h tulostaa ohjeen)"
+
+#: src/install.rs:434 src/install.rs:461 src/search.rs:324 src/search.rs:360 src/lib.rs:227
+msgid " there is nothing to do"
+msgstr " ei mitään tehtävää"
+
+#: src/install.rs:488
+msgid "can't install AUR package as root"
+msgstr "ei pystytty asentamaan AUR pakettia root-käyttäjänä"
+
+#: src/install.rs:507
+msgid "Remove make dependencies after install?"
+msgstr "Poistetaanko make riippuvuudet asennuksen jälkeen?"
+
+#: src/install.rs:518
+msgid "Proceed to review?"
+msgstr "Jatketaanko tarkistukseen?"
+
+#: src/install.rs:559
+msgid "The following packages are not compatible with your architecture:"
+msgstr "Seuraavat paketit eivät ole yhteensopvia arkkitehtuurisi kanssa:"
+
+#: src/install.rs:573
+msgid "Would you like to try build them anyway?"
+msgstr "Haluaisitko kuitenkin kokeilla niiden rakentamista?"
+
+#: src/install.rs:662
+msgid "packages failed to build: {}"
+msgstr "pakettien rakennus epäonnistui: {}"
+
+#: src/install.rs:698
+msgid "could not find .SRINFO for '{}'"
+msgstr "ei pystytty löytämään .SRCINFOa paketille: '{}'"
+
+#: src/install.rs:762
+msgid "failed to read dir: {}"
+msgstr "ei pystytty lukemaan kansiota: {}"
+
+#: src/install.rs:778
+msgid "{} is a directory\n"
+"{} on kansio\n"
+""
+msgstr ""
+
+#: src/install.rs:818
+msgid "failed to open: {}"
+msgstr "ei pystytty avaamaan: {}"
+
+#: src/install.rs:830
+msgid "binary file: {}"
+msgstr "binääri tiedosto: {}"
+
+#: src/install.rs:852
+msgid " nothing new to review"
+msgstr " ei uutta tarkistettavaa"
+
+#: src/install.rs:871
+msgid "failed to execute file manager: {}"
+msgstr "ei pystytty suorittamaan tiedostoselainta: {}"
+
+#: src/install.rs:874
+msgid "file manager did not execute successfully"
+msgstr "tiedostoselain ei suorittunut onnistuneesti"
+
+#: src/install.rs:936
+msgid "duplicate packages: {}"
+msgstr "duplikaatti paketti: {}"
+
+#: src/install.rs:975
+msgid "could not find all required packages:"
+msgstr "ei pystytty löytämään kaikkia vaadittuja paketteja"
+
+#: src/install.rs:982
+msgid "\n"
+"    {:missing} (wanted by: {:stack})"
+msgstr ""
```

Comment:
Not translated.

Reply:
Fixed this in the latest commit. 

I gather the {:text} are variables to be replaced by package names? 

In this context I wasn't able to add in a satisfactory, simple translation so I had left it empty, but this time commit I offered in at least a placeholder.

Reply:
Yes though I realise it's not working so I need to look into that hmmm